StripeIdentity/StripeIdentity/Source/NativeComponents/Detectors/MBDetector/MBCCFrameAnalysisResult+Extensions.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  MBCCaptureState+Extensions.swift
+//  StripeIdentity
+//
+//  Created by Chen Cen on 1/18/24.
+//
+
+import CaptureCore
+import Foundation
+
+extension MBCCFrameAnalysisResult {
+
+    func toDetectorResult() -> MBDetector.DetectorResult {
+        switch self.captureState {
+        case .sideCaptured:
+            // we never have this
+            return .error(.unexpectedSideCaptured)
+        case .documentCaptured:
+            let analyzerResult = MBCCAnalyzerRunner.shared().detachResult()
+            guard
+                let captureResult = analyzerResult.firstCapture,
+                let original = captureResult.capturedImage?.image?.unrotate(),
+                let transformed = captureResult.transformedImage?.image,
+                let side: DocumentSide = captureResult.side == .front ? .front : .back // Per MB's feedback
```

Comment:
DocumentSide only has two values, .front and .back, so why not just have: 
```
let side = captureResult.side
```
Also, all comments in this file are not very helpful.  I'd recommend improving the quality of the comments, or just getting rid of them altogether.

Reply:
oh they are not the same type, `DocumentSide` is defined by us, while `captureResult.side` is `MBCCSideCaptureResultSide`